src/docs/resources/faq.md

```diff
@@ -416,59 +415,65 @@ Hot reload works by injecting updated source code files
 into the running Dart VM (Virtual Machine). This includes
 not only adding new classes, but also adding methods and
 fields to existing classes, and changing existing functions.
-A few types of code changes cannot be hot reloaded though:
+Hot restart resets the state to the app's initial state.
 
-* Global variable initializers.
-* Static field initializers.
-* The `main()` method of the app.
-
-See [Hot reload][] for additional details.
+For more information, see [Hot reload][].
 
 ### Where can I deploy my Flutter app?
 
-You can compile and deploy your Flutter app to iOS and Android.
+You can compile and deploy your Flutter app to iOS, Android,
+[web][] (in beta), and [desktop][] (macOS is in alpha,
+Linux and Windows are in development). While some folks
+are already deploying Flutter web apps,
+you might want to wait for web and desktop support to
+migrate to the stable channel.
 
 ### What devices and OS versions does Flutter run on?
 
-Mobile operating systems: Android Jelly Bean, v16, 4.1.x or newer,
-and iOS 8 or newer.
+* Mobile operating systems: Android Jelly Bean, v16,
+  4.1.x or newer, and iOS 8 or newer.
+
+* Mobile hardware: iOS devices (iPhone 4S or newer)
+  and ARM Android devices.
 
-Mobile hardware: iOS devices (iPhone 4S or newer) and ARM Android devices.
+* Flutter supports building ahead-of-time (AOT) compiled libraries
+  for `x86_64`, `armeabi-v7a`, and `arm64-v8a`.
 
-Note Flutter currently does not support building for x86 Android
-([issue #9253][]) directly, however apps built for ARMv7 or
-ARM64 run fine (via ARM emulation) on many x86 Android devices.
+* Apps built for ARMv7 or ARM64 run fine (using ARM emulation)
+  on many x86 Android devices.
 
-We support developing Flutter apps with Android and iOS devices,
-as well as with Android emulators and the iOS simulator.
+* We support developing Flutter apps with Android and iOS devices,
+  as well as with Android emulators and the iOS simulator.
 
-We test on a variety of low-end to high-end phones
-but we don't yet have an official device compatibility guarantee.
+* We test on a variety of low-end to high-end phones
+  but we don't yet have an official device compatibility guarantee.
 
-We believe Flutter works well on tablets. We do not currently implement
-all of the tablet-specific adaptations recommended by Material Design,
-though we are planning further investment in this area.
+* We believe Flutter works well on tablets. We do not
```

Comment:
We can be stronger than "We _believe_ Flutter works well on tablets". There's nothing fundamentally different about iPadOS or Android on tablets in terms of its compilation / execution, and we test on those devices just like we test on phones.